### ISSUE TITLE
FIRAnalyticsConnector not found

### ISSUE THREAD

**Issue by ashishparmar** [evollu/react-native-fcm#1071]
I have used cocoapods approach 
And my pods looks like :
![screen shot 2018-11-19 at 3 51 04 pm](https://user-images.githubusercontent.com/6857486/48700817-fb086d00-ec12-11e8-9b93-2054e5ae3809.png)
Still I am getting FIRAnalyticsConnector framwork not found

![screen shot 2018-11-19 at 3 52 51 pm](https://user-images.githubusercontent.com/6857486/48700897-32771980-ec13-11e8-888e-b5756ae7a075.png)

**Comment by roots-ai** [comment]
How did you resolve this?

**Comment by ashishparmar** [comment]
@roots-ai yes I have resolved but I can not get push whether app in background or foreground
I can get FCM_Token perfectly but when I try to send via firebase console or from api rest client I just get a success response but not any push in real device 
`{
    "multicast_id": 5795576345818531662,
    "success": 1,
    "failure": 0,
    "canonical_ids": 0,
    "results": [
        {
            "message_id": "0:1542883228414419%d4db7150d4db7150"
        }
    ]
}`

**Comment by Daavidaviid** [comment]
@ashishparmar care to share how you solved that ?

I too have :
```
framework not found FIRAnalyticsConnector
```

**Comment by ashishparmar** [comment]
@Daavidaviid  I have just drag and drop FIRAnalyticsConnector framwork from pods to build and it's works

**Comment by anshumanburman** [comment]
thanks @Daavidaviid . it works for me also

**Comment by ckOfor** [comment]
What does it mean to drag and drop from pods to build can you be more specific @ashishparmar

**Comment by ckOfor** [comment]
I saw a fix by adding $(inherited) to target's build Framework search path

**Comment by ognjetina** [comment]
tried both... nothing works... life has no meaning...

**Comment by javi10823** [comment]
any solution of this?

**Comment by ognjetina** [comment]
so far take a look is your `pod client` updated (not pods in project but pod client on your machine)
add $(inherited) to target's build Framework search path
and do the drag & drop thingy...

**Comment by cgdstnc** [comment]
adding $(inherited) to target's build Framework search path resolved my issue.

**Comment by heyaj19** [comment]
Any solution for this?

**Comment by Myerden** [comment]
I resolved my problem with Sultan Ali's answer: https://stackoverflow.com/a/55025730

**Comment by allthetime** [comment]
I already have $(inherited) in my search path and have added the frameworks in question to my "Linked Framework and Libraries" and I am still getting the same error:

:-1: Framework not found FIRAnalyticsConnector

This is incredibly frustrating I've been trying all day. Maybe its a problem with the new version of Firebase? Can anyone confirm for me what version they are using that is working?

Thanks.

**Comment by lazarevand1997** [comment]
@allthetime you can try to add 
pod 'Firebase/Crash'
dont know why, but works for me

**Comment by allthetime** [comment]
@lazarevand1997 according to pods that lib is outdated. I managed to install it but it didn't help. 

I gave up trying to figure out what it was and just built the ios folder from scratch. It seems there was something weird about my xcode/project settings that was messing thing up.

I deleted my ios folder (after saving a copy):

`rm -rf ios`

created a new one 

`react-native eject`

and then relinked all my other libraries, Firebase first.

**Comment by DiegoResnik** [comment]
this solved this issue for me:

Going into the Targets Build Settings (not the Project), searching for Other Linker Flags, and then deleting FIRAnalyticsConnector from the list took care of the issue for me.

https://stackoverflow.com/questions/64171400/xcode-12-firanalyticsconnector-build-error

**Comment by Coder-ACJHP** [comment]
Today I updated my project pods and faced same issue, then I noticed really there is no FIRAnalyticsConnector framework and I removed the framework name from Build Settings -> Framwork Search Paths (debug and release) and it worked like charm
